### PR TITLE
Lesson 5 task object gauss

### PR DESCRIPTION
**Pull request by ChaikaS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-Internal courses
\ No newline at end of file
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+<<<<<<< HEAD
```
Comment (mikalaider) [inline README.md]: Please remove these. This was part of the merge conflicts

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+Internal courses
+
+# Lesson_4. Curve Gaussian
+
+# Lesson_3. Script operations
+
+
+# Lesson_5. Object Gauss
+>>>>>>> 342afba... add to the file
+>>>>>>> a2730e3... add readme
```
Comment (mikalaider) [inline README.md]: the same

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+
+# Lesson_5. Object Gauss
+
+>>>>>>> 4dd37d8... del to folder and fix readme
```
Comment (mikalaider) [inline README.md]: and the same

Comment (ChaikaS) [reply README.md]: okey, i will fix this

**Pull request merged**